src/hooks/item.ts

```diff
@@ -355,6 +356,33 @@ export default (
         cacheTime: CONSTANT_KEY_CACHE_TIME_MILLISECONDS,
       }),
 
+    useRecycledItems: (options?: { getUpdates?: boolean }) => {
+      const queryClient = useQueryClient();
+      const getUpdates = options?.getUpdates ?? enableWebsocket;
+
+      const { data: currentMember } = useCurrentMember();
+      itemWsHooks?.useRecycledItemsUpdates(
+        getUpdates ? currentMember?.id : null,
+      );
+
+      return useQuery({
+        queryKey: RECYCLED_ITEMS_KEY,
+        queryFn: (): Promise<List<ItemRecord>> =>
+          Api.getRecycledItemsData(queryConfig).then(
+            (data) =>
+              convertJs(data)?.map(({ item }: RecycledItemDataRecord) => item),
+          ),
+        onSuccess: async (items) => {
+          // save items in their own key
+          // eslint-disable-next-line no-unused-expressions
+          items?.forEach(async (item) => {
+            queryClient.setQueryData(buildItemKey(item.id), item);
+          });
```

Comment:
Not sure what this is supposed to do ? I understand that this changes the query cache, and puts the items that were get from the recycle bin into a key ? I do not see an indication that this key is specific to the recycled items, is that right ?

Reply:
Yeah not sure it's necessary since we cannot access trashed items.